### ISSUE TITLE
Exchange a word sent in POST by another?

### ISSUE THREAD

**Issue by viapratica** [typicode/json-server#702]
Hello! Is it possible to exchange a word sent in POST by another? Example: Title: "SERV" and write to db.json Title: "SERVER"

Thank you all and congratulations for the project!

**Comment by xRahul** [comment]
Yes. You can use [Middlewares](https://github.com/typicode/json-server#add-middlewares)
